package.json

```diff
@@ -1,29 +1,40 @@
 {
   "name": "scribe-editor",
-  "version": "3.3.0",
+  "version": "4.0.0",
```

Comment:
Maybe we want to publish Scribe underneath https://www.npmjs.com/org/guardian now?